scripts/engines/boa.sh

```diff
@@ -9,7 +9,8 @@ cd ..
 rm -rf boa
 
 # download release
-curl -L -o boa https://github.com/boa-dev/boa/releases/download/nightly/boa-linux-amd64
+curl -L -o boa https://github.com/boa-dev/boa/releases/download/nightly/boa-nightly-x86_64-unknown-linux-gnu.tar.gz
+tar -xvf boa
```

Comment:
I'd prefer a separate file name here to avoid confusion:

```suggestion
curl -L -o boa.tar.gz https://github.com/boa-dev/boa/releases/download/nightly/boa-nightly-x86_64-unknown-linux-gnu.tar.gz
tar -xvf boa.tar.gz
```